### DISCUSSION TITLE
PHP problem!

### DISCUSSION THREAD

**Discussion by TCloete29** [EddieHubCommunity/support#873] [Q&A] [answered]
Hi 
I have just started a web developing course and am a first time developer so please excuse if I use the wrong terminology. 
In one of my lessons I have to open a new file in Sublime Text and save it as a PHP file. If I save it in Applications -> XAMMP -> htdocs as for example test.php the php code does not change color. So ofcourse when I go to chrome and type in localhost/dashboard/test.php it will not read what is in my string. Is it a directory issue? I have done everything the same as every youtube video and internet guide I can find, but it does not seem to read the code.

**Comment by schmelto** [comment, marked as answer]
- did you start xampp? Your website should start on something like localhost:8080/test.php
- force reload Google Chrome. (you have to Google that one but it's a difference between relaod `F5` and a forced reload) cause styling often goes into the cash of your browser

If this not works may you can share some more details on your problem.

**Reply by TCloete29** [reply]
Yes xampp  is started, proFTD, apache and MySQL running, localhost:8080 enabled. The only reason i didn't have the 8080 before is because of a different way i installed it, but it works the same. As soon as I save my sublime text file as shown in the directory above, the php code  should change color, but it doesn't. Don't know what more details to give :(

**Reply by schmelto** [reply]
do you have the coding somewhere online that I can take a look?

**Reply by TCloete29** [reply]
![Screenshot 2021-02-05 at 15 02 30](https://user-images.githubusercontent.com/78567313/107083350-7db17e80-67c3-11eb-9f07-d3332b5c430e.png)

![Screenshot 2021-02-05 at 15 04 06](https://user-images.githubusercontent.com/78567313/107083379-886c1380-67c3-11eb-860a-26086ceb9a56.png)

![Screenshot 2021-02-05 at 15 04 16](https://user-images.githubusercontent.com/78567313/107083392-8e61f480-67c3-11eb-84d4-8eccd74c5578.png)

![Screenshot 2021-02-05 at 15 03 19](https://user-images.githubusercontent.com/78567313/107083403-9326a880-67c3-11eb-8191-80c732888bb6.png)

**Reply by TCloete29** [reply]
does that help?

**Reply by schmelto** [reply]
Yeah you got the wrong URL. `thinkes-macbook-air.local/test.php` should work.

![image](https://user-images.githubusercontent.com/30869493/107114920-a5f3b880-6869-11eb-8b25-7aa209ae37af.png)

```php
<?php
    echo "Test";
?>
```

![image](https://user-images.githubusercontent.com/30869493/107114973-be63d300-6869-11eb-97c3-dcd722b070f1.png)

`localhost/...` is the root htdocs folder.

`localhost/dashboard` it will look into `htdocs/dashboard/...` for your file.

If you want to keep the dashboard in your link you can move the `test.php` file into the dashboard folder in htdocs.

![image](https://user-images.githubusercontent.com/30869493/107115077-65e10580-686a-11eb-9830-d8f9e8149d86.png)

![image](https://user-images.githubusercontent.com/30869493/107115081-6d081380-686a-11eb-9404-c78d52fc829d.png)

Hope this was the issue.

**Reply by TCloete29** [reply]
It works! Agh so simple lol. THANK YOU!

**Reply by schmelto** [reply]
> It works! Agh so simple lol. THANK YOU!

You're welcome!

**Reply by eddiejaoude** [reply]
Great team work 👍
